### ISSUE TITLE
Add pgcat

### ISSUE THREAD

**Issue by kingluo** [dhamaniasad/awesome-postgres#306]
pgcat - Enhanced postgresql logical replication

https://github.com/kingluo/pgcat

**Comment by kmoppel** [comment]
@kingluo Hi, is the project active? Please create a PR if so...

**Comment by kingluo** [comment]
@kmoppel yes, it's active. Please see the PR: https://github.com/dhamaniasad/awesome-postgres/pull/429
